app/helpers/syntax_helper.rb

```diff
@@ -25,4 +27,9 @@ def vue_component(identifier, variable=nil)
       }; nil
     end
   end
+
+  #server side rendering via hypernova
+  def render_vue(id, name)
```

Comment:
💡 keep the DSL and parameter consistent to `hypernova` helper?

```rb
def render_vue_component(id, name: name)
  ...
end
```

Ruby has an `alias_method` feature which could be proposed to `hypernova`

Reply:
Took your suggestion https://github.com/vueonrails/vueonrails/commit/5a28a6f6c052f64efd7352dc46f4e9c15b0c082b

Thanks 🙏